node_common/utilities.js

```diff
@@ -140,9 +140,8 @@ export const refreshLibrary = async ({ state, PG, FFS }) => {
         }
 
         PG.ffs.watchJobs((job) => {
-          // NOTE(jim): FFS is undefined?
           console.log("[ prototype ] job status", job.status);
-          if (job.status >= 5) {
+          if (job.status === FFS.JobStatus.JOB_STATUS_SUCCESS) {
```

Comment:
Please revert this back to whats on the other side. `if (job.status >= 5) {`

Reply:
All set w/ latest commit.